cobspec/src/main/java/com/saralein/cobspec/controller/LogController.java

```diff
@@ -0,0 +1,59 @@
+package com.saralein.cobspec.controller;
+
+import com.saralein.cobspec.data.LogStore;
+import com.saralein.server.controller.Controller;
+import com.saralein.server.request.Request;
+import com.saralein.server.response.Response;
+import java.util.Base64;
+
+public class LogController implements Controller {
+    private final LogStore logStore;
+    private final String username;
+    private final String password;
+
+    public LogController(LogStore logStore, String username, String password) {
+        this.username = username;
+        this.password = password;
+        this.logStore = logStore;
+    }
+
+    public Response respond(Request request) {
+        return isAuthorized(request) ? createAuthorizedResponse() : createUnAuthorizedResponse();
+    }
+
+    private Response createAuthorizedResponse() {
+        System.out.print(logStore.retrieveLog());
```

Comment:
Shouldn't we be using a logger abstraction instead of system.out 🙃

Reply:
Removed now.  It was a test line I forgot about.